app/src/main/java/net/gsantner/markor/ui/hleditor/HighlightingEditor.java

```diff
@@ -102,6 +107,57 @@ public void beforeTextChanged(CharSequence s, int start, int count, int after) {
                 }
             }
         });
+
+        addTextChangedListener(new TextWatcher() {
+            @Override
+            public void beforeTextChanged(CharSequence s, int start, int count, int after) {
+                return;
+            }
+
+            @Override
+            public void afterTextChanged(Editable e) {
+                Spannable eSpan = (Spannable) e;
+                for (Object span : eSpan.getSpans(0, e.length(), this.getClass())) {
+                    if ((eSpan.getSpanFlags(span) & Spanned.SPAN_COMPOSING) != 0) {
+                        e.delete(eSpan.getSpanStart(span), eSpan.getSpanEnd(span));
+                    }
+                }
+            }
+
+            @Override
+            public void onTextChanged(CharSequence s, int start, int before, int count) {
+
+                // This method is called to notify you that, within s,
+                // the count characters beginning at start have just replaced old text that had length before.
+                if ( count > 0 && start < s.length() && s.charAt(start) == '\n' ) {
+                    // Check if previous like matches
+                    int iStart, iEnd;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use one line for each declaration, it enhances code readability.](https://app.codacy.com/manual/gsantner/markor/pullRequest?prid=5254231)